dataeng/resources/snowflake-refresh-snowpipe.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup
+cd $WORKSPACE/analytics-tools/snowflake
+make requirements
+
+python refresh_snowpipe.py -k $KEY_PATH -s $PASSPHRASE -p $PIPE_NAME -t $TABLE_NAME -d $DELAY
```

Comment:
Assuming that you'll implement the agreed change to move passphrase into a file.